README.md

```diff
@@ -10,9 +10,10 @@ CompStore web store frontend
     -   [Repositories](#repositories)
     -   [Commit prefixes](#commit-prefixes)
     -   [Name conventions](#name-conventions)
-    -   [CSS class](#css-class)
-    -   [Types](#types)
+        -   [CSS class](#css-class)
```

Comment:
Chyba coś źle sformatowało te linie

Reply:
A czemu? Nie podałeś wskazówki co jest nie tak, więc za bardzo nie wiem o co chodzi.

Reply:
Zasugerowałem się tym przesunięciem, lecz CSS Class to podpunkt spisu treści, więc powinien być tak na prawo. Jest OK